media/js/src/assetDetail/ViewItem.jsx

```diff
@@ -34,34 +34,75 @@ export default class ViewItem extends React.Component {
             userIsAuthor = authorId === window.MediaThread.current_user;
         }
 
+        const references = [];
         if (this.state.references) {
             this.state.references.forEach(function(reference, idx) {
                 references.push(
-                    <div className="card w-100 card-selection" key={idx}>
-                        <div className="card-body">
-                            <h5 className="card-title">
-                                <div>
-                                    {reference.title}
-                                </div>
-                            </h5>
-                            <h6>
-                                Assignment
-                            </h6>
-                            <p className="card-text">
-                                Collected 12/19/15 01:02 AM
-                            </p>
-                            <a href="#" className="btn btn-primary btn-sm">
-                                View
-                            </a>
-                        </div>
-                    </div>
+                    <li key={idx}>
+                        <a href={reference.url}>{reference.title}</a>
+                    </li>
                 );
             });
         }
 
+        let tags = '(None)';
+        let terms = '(None)';
+        if (this.props.asset && this.props.asset.annotations) {
+            const tagsArray = getTags(this.props.asset.annotations);
+            if (tagsArray.length > 0) {
+                tags = tagsArray.join(', ');
+            }
+
+            const termsArray = getTerms(this.props.asset.annotations);
+            if (termsArray.length > 0) {
+                terms = termsArray.join(', ');
+            }
+        }
+
+        let description = null;
+        if (this.props.asset && this.props.asset.metadata) {
+            let desc = find(this.props.asset.metadata, {key: 'Description'});
+            if (desc) {
+                desc = desc.value;
+            }
+            if (desc && desc.length) {
+                description = desc[0];
+            }
+        }
+
         return (
             <div className="tab-content" id="pills-tabContent">
-                <table className="table mt-1">
+                <h3>Tags</h3>
+                <p>
+                    {tags}
+                </p>
+
+                <hr />
+
+                <h3>Course Vocabulary (Terms)</h3>
```

Comment:
We always call this "terms" in the interface. Until the team as a whole renames this feature, can the title just say "Terms" for consistency?